recipes/linkage2allegro/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: linkage2allegro
+  version: "2017.1"
+
+source:
+  url: https://github.com/BioTools-Tek/linkage-converter/archive/d8eefb76ad2cf3ab5fb6b190e968f6f933cc2886.tar.gz
+  md5: 64766aff1842c1f36ae1d00ccf9dbd07
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - future
```

Comment:
Most likely the `future` dependency isn't needed. Things like `from __future__ import print_function` are already covered by the Python standard library itself